pkg/test/resourcefixture/testdata/basic/run/v1beta1/runjob/jobwithkmscryptokey/_http.log

```diff
@@ -571,21 +572,21 @@ X-Xss-Protection: 0
 
 ---
 
-POST https://run.googleapis.com/v2/projects/${projectId}/locations/us-central1/jobs?alt=json&jobId=${jobID}
+POST https://run.googleapis.com/v2/projects/${projectId}/locations/us-central1/jobs?%24alt=json%3Benum-encoding%3Dint&jobId=${jobID}
 Content-Type: application/json
 User-Agent: kcc/${kccVersion} (+https://github.com/GoogleCloudPlatform/k8s-config-connector) kcc/controller-manager/${kccVersion}
+X-Goog-Request-Params: location=us-central1
 
 {
-  "launchStage": "GA",
+  "launchStage": 4,
   "template": {
     "template": {
       "containers": [
         {
           "image": "us-docker.pkg.dev/cloudrun/container/hello"
         }
       ],
-      "encryptionKey": "projects/${projectId}/locations/us-central1/keyRings/kmskeyring-${uniqueId}/cryptoKeys/kmscryptokey-1-${uniqueId}",
-      "maxRetries": 3
```

Comment:
ditto: TF-based adds a fixed value maxRetries to the HTTP requests, this is the same value as GCP defaults to if not specified.